### ISSUE TITLE
Platform with type "Windows_NT" and architecture "x64" is not supported by @bugsnag/cli

### ISSUE THREAD

**Issue by goblinbr** [bugsnag/bugsnag-cli#93]
on `npm install --save-dev @bugsnag/cli` it's always showing this error:
```
npm ERR! code 1
npm ERR! path C:\projetos\AplicativoAgronomos\node_modules\@bugsnag\cli
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c node install.js
npm ERR! Platform with type "Windows_NT" and architecture "x64" is not supported by @bugsnag/cli.
npm ERR! Your system must be one of the following:
npm ERR!
npm ERR! [
npm ERR!   {
npm ERR!     "Type": "Windows",
npm ERR!     "Architecture": "x64",
npm ERR!     "Artifact": "x86_64-windows-bugsnag-cli.exe"
npm ERR!   },
npm ERR!   {
npm ERR!     "Type": "Windows",
npm ERR!     "Architecture": "i386",
npm ERR!     "Artifact": "i386-windows-bugsnag-cli.exe"
npm ERR!   },
npm ERR!   {
npm ERR!     "Type": "Linux",
npm ERR!     "Architecture": "x64",
npm ERR!     "Artifact": "x86_64-linux-bugsnag-cli"
npm ERR!   },
npm ERR!   {
npm ERR!     "Type": "Linux",
npm ERR!     "Architecture": "i386",
npm ERR!     "Artifact": "i386-linux-bugsnag-cli"
npm ERR!   },
npm ERR!   {
npm ERR!     "Type": "Darwin",
npm ERR!     "Architecture": "x64",
npm ERR!     "Artifact": "x86_64-macos-bugsnag-cli"
npm ERR!   },
npm ERR!   {
npm ERR!     "Type": "Darwin",
npm ERR!     "Architecture": "arm64",
npm ERR!     "Artifact": "arm64-macos-bugsnag-cli"
npm ERR!   }
npm ERR! ]

npm ERR! A complete log of this run can be found in: C:\Users\Rodrigo\AppData\Local\npm-cache\_logs\2024-04-11T12_30_34_219Z-debug-0.log
```

Node version: v18.20.1
NPM version: 10.5.2
OS: Windows 11 (23H2 22631.3447)

[2024-04-11T12_30_34_219Z-debug-0.log](https://github.com/bugsnag/bugsnag-cli/files/14946416/2024-04-11T12_30_34_219Z-debug-0.log)

**Comment by mclack** [comment]
Hi @goblinbr 

Thanks for bringing our attention to this, and for raising the associated [PR](https://github.com/bugsnag/bugsnag-cli/pull/94). Our CI has a restriction on allowing externally raised PRs, so we will need to take your commit and add it into one of our own PRs, but you will still be credited for this.

Are you able to confirm that the BugSnag CLI works as intended on your machine with the proposed changes?

**Comment by goblinbr** [comment]
yes, it's working

```
npm install --save-dev @bugsnag/cli@git://github.com/goblinbr/bugsnag-cli.git#cf1f418879594dc4fca6832cfde396c3942bb537
npm WARN skipping integrity check for git dependency ssh://git@github.com/goblinbr/bugsnag-cli.git

added 12 packages, and audited 13 packages in 10s

1 package is looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

```
dir node_modules\.bin
11/04/2024  14:13    <DIR>          .
11/04/2024  14:13    <DIR>          ..
11/04/2024  14:13         9.573.888 bugsnag-cli.exe
11/04/2024  14:13               393 js-yaml
11/04/2024  14:13               327 js-yaml.cmd
11/04/2024  14:13               813 js-yaml.ps1
               4 arquivo(s)      9.575.421 bytes
               2 pasta(s)   280.576.692.224 bytes disponíveis
```

**Comment by toniewypada** [comment]
Hello :wave: ,

Your PR has been reviewed and included in release [v2.2.0 do BugSnag CLI](https://github.com/bugsnag/bugsnag-cli/releases/tag/v2.2.0). Thanks for your contribution!

If you have any questions, please let us know!
